src/debug/HyprCtl.cpp

```diff
@@ -1374,73 +1374,65 @@ std::string decorationRequest(eHyprCtlOutputFormat format, std::string request)
     return result;
 }
 
-void createOutputIter(wlr_backend* backend, void* data) {
-    const auto DATA = (std::pair<std::string, bool>*)data;
+static bool addOutput(wlr_backend* backend, const std::string& type, std::string& name) {
+    wlr_output* output = nullptr;
 
-    if (DATA->second)
-        return;
-
-    if (DATA->first.empty() || DATA->first == "auto") {
-        if (wlr_backend_is_wl(backend)) {
-            wlr_wl_output_create(backend);
-            DATA->second = true;
-        } else if (wlr_backend_is_x11(backend)) {
+    if (type.empty() || type == "auto") {
+        if (wlr_backend_is_wl(backend))
+            output = wlr_wl_output_create(backend);
+        else if (wlr_backend_is_x11(backend))
```

Comment:
while we're at it, we can nuke all x11 things (disabled at comptime anyways)

Reply:
done